library/stdlib/qsort_r.c

```diff
@@ -68,24 +68,28 @@ pntz(size_t p[2]) {
 
 static void
 cycle(size_t width, unsigned char *ar[], int n) {
-    unsigned char tmp[256];
+    size_t tmp_size = 256;
+    unsigned char *tmp = malloc(tmp_size);
     size_t l;
     int i;
 
     if (n < 2) {
+        free(tmp);
         return;
     }
 
     ar[n] = tmp;

```

Comment:
With these changes the following critical error is fixed, referring to this line.
> Address of local auto-variable assigned to a function parameter.